src/frontend/components/PublicVideoDashboard/index.tsx

```diff
@@ -35,10 +35,19 @@ const PublicVideoDashboard = ({
   const timedTextTracks = useTimedTextTrack((state) =>
     state.getTimedTextTracks(),
   );
+  const displayJoinDiscussionAskButton = useMemo(
```

Comment:
I'm not sur memo here is mandatory since computation duration is not an issue here, you can use a local variable

Reply:
@jbpenrath what do you think ? I made this modification on your advice, do we keep `useMemo` or not ?

Reply:
If I may, I think we are in the same situation than [this comment](https://github.com/openfun/marsha/pull/1216#discussion_r755021476)

if the first comment was [this comment](https://github.com/openfun/marsha/pull/1216#discussion_r753176160), I think this was more about declaring a local variable since this is not CPU costly and we don't need to prevent this value to be recalculated

Reply:
PS : I need to open my eyes sometimes, code suggestion was with a memo

Reply:
I have removed this change. Thanks both for your feedback :+1: